etc/apps.rc

```diff
@@ -17,22 +19,3 @@ medium:
 large:
 
 xlarge:
-
```

Comment:
We need that profile for update-dependencies. Otherwise it does not know application modules.
If we have another way to iterate through modules (keywords?) we can live without this profile